### ISSUE TITLE
warning in gimi 7 not printing out

### ISSUE THREAD

**Issue by JaxSie** [SilentNightSound/GI-Model-Importer#110]
as the title said, theres a "tut" sound everytime i launch the game or reload, n theres no warning /error log like previous gimi version. is there a way to turn it on? 
ps i know which mod is at fault but dunno what is causing it..

**Comment by DocZeus** [comment]
same question but I don't have any fault with mods(so far I know). I would still like to get to the bottom of the sound.

**Comment by leotorrez** [comment]
hunting = 1 in d3dx.ini should show dev console

**Comment by DocZeus** [comment]
yes, thanks, found that day before yesterday xD, been solving my errors since then.

**Comment by Gattea** [comment]
I was glad to find this question, as I'm having the same problem, however, changing the above value still doesn't print out warnings on my end.  Would anyone happen to be able to guess why?

**Comment by DocZeus** [comment]
> I was glad to find this question, as I'm having the same problem, however, changing the above value still doesn't print out warnings on my end. Would anyone happen to be able to guess why?

install the dev version of GIMI
